ministryofjustice/analytics-platform-auth-proxy
# Show error message on failed login

## What

* Get the Auth0 error message out of the callback URL parameters and pass to Auth0 Lock to display to user

## How to test

* Add a customer to an App A in the control-panel
* Submit the customer email in the login page for App B where the customer is not whitelisted
* Receive a login email and click through
* See the login page has an error message "UNAUTHORIZED"
